File: kyw0716/2주차/첫 시도/콜라츠 추측.js
function solution(num) {
  var answer = 0;
  while (answer < 500) {
    if (num === 1) break;
    if (num % 2 === 0) num /= 2;
    else {
      num = num * 3 + 1;
    }
    answer += 1;
  }
  return answer > 499 ? -1 : answer;
}

File: sk000801/콜라츠 추측.java
class Solution {
    public int solution(int num) {
        int count = 0;
        while(num != 1) {
            if(count>500) {
                count = -1;
                break;
            }    
            if(num % 2 == 0) {
                num /= 2;
            }
            else if(num % 2 == 1) {
                num = 3*num+1;
            }
            count ++;
        }
        return count;
    }
}